app/views/organisations/show.html.erb

```diff
@@ -33,7 +33,7 @@
     <% unless @organisation.is_promotional_org? %>
       <%= render partial: 'freedom_of_information' %>
     <% end %>
-    <% if @show.high_profile_groups[:items] && !@organisation.is_promotional_org? %>
+    <% if @show.high_profile_groups[:items]&.any? && !@organisation.is_promotional_org? %>
```

Comment:
We could used `present?` here as it takes into account `nil` so you don't need the `&`, which is slightly nicer to read.

Reply:
ah very good, thank you!